Tabloid/client/src/components/Tag/tagForm.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from "react";
+import { useHistory } from "react-router-dom";
+import { FormGroup, Input, Label, Button, Form } from "reactstrap";
+import { addTag } from "../../modules/tagManager";
+
+const TagForm = () => {
+    const history = useHistory();
+    const emptyTag = {
+        Name: "",
+    };
+
+    const [tag, setTag] = useState(emptyTag);
+
+    const handleInputChange = (evt) => {
+        const value = evt.target.value;
+        const key = evt.target.id;
+
+        const tagCopy = { ...tag };
+
+        tagCopy[key] = value;
+        setTag(tagCopy);
+    };
+    const handleSave = (evt) => {
+        evt.preventDefault();
+        addTag(tag).then((p) => {
+            history.push("/tags");
+        });
+    };
+
+    return (
+        <Form>
+            <FormGroup>
+                <Label for="name">Name</Label>
+                <Input type="text" name="name" id="name" placeholder="tag name" value={tag.name} onChange={handleInputChange} />
+            </FormGroup>
+            <Button className="btn btn-primary" onClick={handleSave}>Submit</Button>
```

Comment:
Consider adding some kind of form validation here to disable the button until the user has filled out the form input completely.